plugin/src/main/java/lol/pyr/znpcsplus/parsers/StringParser.java

```diff
@@ -14,6 +14,6 @@ public StringParser(Message<CommandContext> message) {
 
     @Override
     public String parse(Deque<String> deque) throws CommandExecutionException {
-        return deque.pollFirst();
+        return String.join(" ", deque);
```

Comment:
Why do we even need this class? `CommandContext#popString()` and `CommandContext#dumpAllArgs()` exists

Reply:
It was for the display name property.

Reply:
Oh right I forgot we do dynamic parsing silly me